ferrocene/doc/specification/.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     # 2022-11-24. See https://github.com/actions/setup-python/issues/544.
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
this one should be done in ferrocene/specification

Reply:
Good point.

But just trying to wrap my head around it: Since it is a subtree, couldn't we also make the change here and push it to ferrocene/specification?

Reply:
that can be, but we don't have automation for that (and we don't want to have to remember to do the push)